docs/docs/routing.md

```diff
@@ -0,0 +1,241 @@
+---
+id: routing
+title: Routing
+permalink: docs/routing.html
+---
+
+Routing is the process of choosing what to render based on the browser's current URL.
+
+The simplest way to implement Routing in a React application is with a top-level Component that chooses which content to render by URL. For example, a top-level `App` component may decide what to render based on [window.location.pathname](https://developer.mozilla.org/en-US/docs/Web/API/Location):
+
+```js
+function Menu() {
+  return (
+    <div>
+      <a href="/news">News</a>
+      <a href="/profile">Profile</a>
+    </div>
+  );
+}
+
+class App extends React.Component {
+  render() {
+    const location = window.location;
+  
+    let content;
+    if (location.pathname == '/news') {
+      content = <h1>News</h1>;
+    }
+    else if (location.pathname == '/profile') {
+      content = <h1>Profile</h1>;
+    }
+    else {
+      content = <h1>Not Found</h1>;
+    }
+    
+    return (
+      <div>
+        <Menu />
+        <div>{content}</div>
+      </div>
+    );
+  }
+}
+```
```

Comment:
Ooh okay. Let's make sure the initial code snippet is copy-pasteable into Create React App without changes then. Like in RR4 docs.